src/app/api/Mock/MockApi.ts

```diff
@@ -271,18 +322,19 @@ export default class MockClient implements API {
         return shiftToAdd as Shift;
     }
 
-    async deleteShift(shiftId: IdType): Promise<void> {
+    async deleteShift(shiftIds: IdType[]): Promise<void> {
```

Comment:
would likely want to give it a default value and check for length greater than 0